internal/provisioner/objects/dataplane/dataplane_test.go

```diff
@@ -154,6 +154,9 @@ func checkDaemonSecurityContext(t *testing.T, ds *appsv1.DaemonSet) {
 func TestDesiredDaemonSet(t *testing.T) {
 	name := "ds-test"
 	cntr := model.Default(fmt.Sprintf("%s-ns", name), name)
+	cntr.Spec.ResourceLabels = map[string]string{
```

Comment:
we can do this in a follow up, but I don't think this line is really testing anything: https://github.com/projectcontour/contour/blob/6fc0daecce9df626085a92872791440889a0e887/internal/provisioner/objects/deployment/deployment_test.go#L143, its just checking if the labels on the resource match against itself...

we should probably write out explicitly what labels we expect

for this and the deployment test